.gitlab-ci/stages/manual.yml

```diff
@@ -49,9 +49,9 @@ Production:
       # that only bump version / udpate changelog
       - $CI_COMMIT_MESSAGE =~ /^chore(release):*/
   script:
-    - echo "deploy ${CI_COMMIT_REF_SLUG}"
+    - echo "deploy ${COMMIT_TAG}"
```

Comment:
I think it's `CI_COMMIT_TAG`, cf: https://docs.gitlab.com/ee/ci/variables/predefined_variables.html

Reply:
good catch ! (COMMIT_TAG exist only after env.sh have run)